setup.py

```diff
@@ -26,6 +26,8 @@
     'rsa>=3.1.4,<5; python_version >= "3"',
     "setuptools>=40.3.0",
     "six>=1.9.0",
+    "aiohttp>=3.6.2; python_version>='3.5'",
```

Comment:
is this 3.5 truly? I know there are some reasons to do 3.6+ only for async

Reply:
Resolved, changed to 3.6+